homeassistant/components/tibber/sensor.py

```diff
@@ -201,36 +213,36 @@ async def _async_callback(self, payload):
         self.async_write_ha_state()
 
     @property
-    def available(self):
+    def available(self) -> Any:
```

Comment:
```suggestion
    def available(self) -> bool:
```

Reply:
Would require typing to be added to the Tibber library?

Reply:
I think so.